frontend/src/concepts/__tests__/StartNotebookModal.spec.tsx

```diff
@@ -5,52 +5,54 @@ import StartNotebookModal from '~/concepts/notebooks/StartNotebookModal';
 import {
   mockCompletedStates,
   mockFailedStates,
-  mockInitialStates,
+  //mockInitialStates,
```

Comment:
You can just remove this line. Your IDE will probably yell at you when you uncomment everything to bring this import back.